site/docs/master/restic.md

```diff
@@ -337,10 +342,19 @@ within each restored volume, under `.velero`, whose name is the UID of the Veler
 1. Once all such files are found, the init container's process terminates successfully and the pod moves
 on to running other init containers/the main containers.
 
+## 3rd party controller
+
+### Monitor backup annotation
+
+If you need a way to check if a volume is not backed up, because a user missed to configure the annotation, at the moment
```

Comment:
Wordsmithing: let's say something like "Velero does not currently provide a mechanism to detect persistent volume claims that are missing the restic backup annotation."

Reply:
Thanks for this.
I updated our fork ;)